tools/internal_ci/linux/grpc_psm_performance_gke_experiment.sh

```diff
@@ -131,14 +131,12 @@ time ../test-infra/bin/prepare_prebuilt_workers \
 
 # Run tests.
 time ../test-infra/bin/runner \
-    -i "psm_${PSM_PROXIED_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_8CORE}.yaml" \
-    -i "psm_${PSM_PROXYLESS_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_8CORE}.yaml" \
-    -i "psm_${PSM_PROXIED_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_32CORE}.yaml" \
-    -i "psm_${PSM_PROXYLESS_TEST}_loadtest_with_prebuilt_workers_${WORKER_POOL_32CORE}.yaml" \
+    -i "psm_proxied_loadtest_with_prebuilt_workers_${WORKER_POOL_8CORE}.yaml" \
+    -i "psm_proxyless_loadtest_with_prebuilt_workers_${WORKER_POOL_8CORE}.yaml" \
```

Comment:
Assuming we are only running on 8 cores, LGTM.

Reply:
8 core should be fine as long as we're not saturating the server.